packages/snippetz/package.json

```diff
@@ -228,6 +228,7 @@
   "devDependencies": {
     "@scalar/build-tooling": "workspace:*",
     "@types/har-format": "^1.2.15",
-    "vite": "^5.4.10"
+    "vite": "^5.4.10",
+    "vitest": "^1.6.0"
```

Comment:
do we need this here? should be available through the workspace already, or am I wrong? 🤔

Reply:
i guess we don't if we run it from root but thought if someone would use the package they might want it - i removed it